test/server_test.py

```diff
@@ -39,11 +39,11 @@
 port = 0
 
 [local]
-socket =
+socket = {run_dir}/local.sock
 
 [control]
 transport = unix
-socket = {run_dir}/sock
+socket = {run_dir}/control.sock
```

Comment:
This is good change regardless of using a local socket, since it is more clear that the socket is the control socket.

But this change is not related to support initial ticket, please split it to another pr to make this pr easier to review.

Reply:
Removed from the PR.